src/main/java/org/jala/efeeder/tyche/TycheCommand.java

```diff
@@ -106,7 +109,7 @@ public static String drawLot(int feastId, int userId, Connection connection) thr
 			dice = diceResult[0];
 			if (dice == -1)
 			{
-				dice = 1 + (int)Math.floor(secureRandom.nextDouble() * Integer.MAX_VALUE);
+				dice = rand.nextInt((999999999 - 100000000) + 1) + 1;
```

Comment:
Please use constants instead of magic numbers. I think the (999999999 - 100000000) + 1 can be a constant value